src/R/Components/Impl/ConnectionManager/IConnection.cs

```diff
@@ -8,8 +8,9 @@ namespace Microsoft.R.Components.ConnectionManager {
     public interface IConnection {
         Uri Id { get; }
         string Name { get; }
+        string Path { get; }
```

Comment:
It looks like `Path` is always an `Uri` internally, too - any reason to not expose it on the interface?